From 6d22c3384373afd628cc8d9c03f6574428aeef15 Mon Sep 17 00:00:00 2001
From: Rob Clark <rclark@redhat.com>
Date: Wed, 17 Dec 2014 21:38:16 -0500
Subject: [drm] nouveau: Bump version from 1.1.1 to 1.1.2

Message-id: <1418852380-13061-6-git-send-email-rclark@redhat.com>
Patchwork-id: 102141
O-Subject: [RHEL7 drm 05/89] drm/nouveau: Bump version from 1.1.1 to 1.1.2
Bugzilla: 1173317
RH-Acked-by: Jerome Glisse <jglisse@redhat.com>
RH-Acked-by: Dave Airlie <airlied@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1173317
Upstream: since v3.16.3

commit b80e628657c501dade91cebbf0ce0466702e3b70

Author:     Mario Kleiner <mario.kleiner.de@gmail.com>
AuthorDate: Wed Aug 6 06:09:44 2014 +0200
Commit:     Rob Clark <rclark@redhat.com>
CommitDate: Wed Dec 10 17:04:25 2014 -0500

    drm/nouveau: Bump version from 1.1.1 to 1.1.2

    commit 7820e5eef0faa4a5e10834296680827f7ce78a89 upstream.

    Linux 3.16 fixed multiple bugs in kms pageflip completion events
    and timestamping, which were originally introduced in Linux 3.13.

    These fixes have been backported to all stable kernels since 3.13.

    However, the userspace nouveau-ddx needs to be aware if it is
    running on a kernel on which these bugs are fixed, or not.

    Bump the patchlevel of the drm driver version to signal this,
    so backporting this patch to stable 3.13+ kernels will give the
    ddx the required info.

    Signed-off-by: Mario Kleiner <mario.kleiner.de@gmail.com>
    Signed-off-by: Ben Skeggs <bskeggs@redhat.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Signed-off-by: Rob Clark <rclark@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/gpu/drm/nouveau/nouveau_drm.h b/drivers/gpu/drm/nouveau/nouveau_drm.h
index 7efbafa..b628add 100644
--- a/drivers/gpu/drm/nouveau/nouveau_drm.h
+++ b/drivers/gpu/drm/nouveau/nouveau_drm.h
@@ -10,7 +10,7 @@
 
 #define DRIVER_MAJOR  1
 #define DRIVER_MINOR  1
-#define DRIVER_PATCHLEVEL 1
+#define DRIVER_PATCHLEVEL 2
 
 /*
  * 1.1.1:
@@ -21,6 +21,8 @@
  *        to control registers on the MPs to enable performance counters,
  *        and to control the warp error enable mask (OpenGL requires out of
  *        bounds access to local memory to be silently ignored / return 0).
+ * 1.1.2:
+ *      - fixes multiple bugs in flip completion events and timestamping
  */
 
 #include <core/client.h>
-- 
1.7.1